src/backend/chat_service/chat_service.py

```diff
@@ -270,5 +270,7 @@ async def check_server_status(request: Request):
 # Run the server
 if __name__ == "__main__":
     import uvicorn
+    import os
 
-    uvicorn.run(app, host="0.0.0.0", port=8000)
+    port = int(os.environ.get("CHAT_SERVICE_PORT", 8000))
```

Comment:
Convert this line without the default value so that we are sure the values are picked up from environmental variables
port = int(os.environ.get("CHAT_SERVICE_PORT", 8000))

Reply:
I replaced the default values with environmental variables.